scripts/ontology_to_ontology_mapping_tool/ontology_mappings.txt

```diff
@@ -0,0 +1,574 @@
+ONCOTREE_CODE	NCIT_CODE	UMLS_CODE	HEMEONC_CODE	ICDO_CODE
```

Comment:
I've scanned through this mappings file and I don't think there are any values for the ICDO_CODE column. Is that intentional?

Reply:
@sheridancbio yes. @ritikakundra will update the mappings soon as we get them.